app/code/core/Mage/Adminhtml/Block/Catalog/Product/Edit/Tab/Websites.php

```diff
@@ -42,7 +42,7 @@ public function getProduct()
     /**
      * Get store ID of current product
      *
-     * @return int
+     * @return string
```

Comment:
https://github.com/OpenMage/magento-lts/blob/df37dba44808847ccbb90a4ec8a53cb6065e832c/app/code/core/Mage/Catalog/Model/Product.php#L215-L215
```suggestion
     * @return int|string
```

Reply:
I think i have xdebuged it, and it is always a string.
 
 The method annotation should be changed. (?)

Reply:
Consider 
```php
$product = Mage::getModel('catalog/product')
       ->setStoreId($storeId)
       ->load($id);
``` 
OpenMage is not type strict: `$storeId` and `$id` can be type string or int. However, the column `store_id` is type `smallint(5)` in the EAV tables, so `int` is more correct in docblock. Therefore, I suggest `int` first then `string`. We can leave the annotation as is.

Reply:
> However, the column store_id is type smallint(5) in the EAV tables, so int is more correct in docblock.

Thats true. Mhhh ... if we want to reflect the DB column type we should turn it into "int" - everywhere.

Reply:
Reverted some changes and cast to int (in a few places).

Reply:
@kiatng any objections to cast store id to int?

Reply:
No objection to cast store ID to int.